index.html

```diff
@@ -11,18 +11,27 @@
 <link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet">
   </head>
   <body>
+    <header>
       <div class="AppBar">
-        <header class="ToolBar">
+        <div class="ToolBar">
           <nav>
             <img src="images/NormalButton.png" alt="" class="NormalButton" />
           </nav>
           <div class="Logo">
             <a href="#">Faishar Faike</a>
           </div>
-        </header>
+          <ul class="nav-menu">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#About">About</a></li>
+       </ul>
+        </div>
```

Comment:
- I think it will be great to make the content in our nav bar a bit more centered just like on figma.

Here is what I can see compared to figma 👇🏽 
### Your navbar

![navbad](https://user-images.githubusercontent.com/99246359/200112203-432464cc-564d-4749-81a8-d526b46481a2.PNG)

### Figma

![corrfig](https://user-images.githubusercontent.com/99246359/200112221-cf12f734-1af4-42a5-895a-34f667c6e7d3.PNG)

- I think you h=may have styled the ul in a `.nav-menu` style in your CSS, Kindly fix it and the logo as well 🌸

Reply:
thanks for observation am going to consider it